### ISSUE TITLE
Standardization?

### ISSUE THREAD

**Issue by flonkler** [0xcf843ecf802c722f434d56/InupiaqNumbers#1]
Is there any standardization that specifies how the strokes should be drawn (angle, length, position)? If not, what constraints did you choose to "draw" these numerals?

**Comment by 0xcf843ecf802c722f434d56** [comment]
There are no standards that I'm aware of, however if you open `InupiaqNumbers.sfd` in [FontForge](https://fontforge.org/), you'll see the lines I used to draw them.

Basically I had a margin of 100 pixels for every glyph. Including the margin, every glyph is 700x1000 pixels. Each 5-mark is 100 pixels tall and as wide as the glyph. Each 1-mark divides the character evenly based on how many there are. The first 1-mark reaches from the last 5-mark (or the top of the glyph for numbers `1-4`) and the however all other 1-marks' top is pushed down by 100 pixels in order to make sure it wouldn't intersect nor get to close to the 5-marks. After applying these rules, the line was extended by 100 pixels, so the glyph actually extends past the margin.

![image showing rules used](https://i.imgur.com/zNdhlrA.png)

For the 0-glyph, I decided to just do a basic path and extend it. I tried to make it as symmetrical as possible, but there's no constraints or rules behind its design.
